src/transition.js

```diff
@@ -76,13 +76,13 @@ export function withCurve(transition, curve) {
  * a specified as a map from CSS property names to transition functions for
  * each of these properties.
  * @param {!Element} element
- * @param {!Object<string, !TransitionDef>} styles
- * @return {!TransitionDef<void>}
+ * @param {!Object<string, !TimeTransitionDef>} styles
+ * @return {!TimeTransitionDef<void>}
```

Comment:
@dvoytenko originally it was only `TransitionDef`, but the codebase sort of had a 50/50 usage of the `complete` argument.

the latter suggestion seems to be a larger refactor change that i definitely don't want to do here